README.md

```diff
@@ -1,54 +1,210 @@
+[commercetools PHP-SDK](https://github.com/commercetools/commercetools-php-sdk) when implementing
+a Symfony project.
+
+##### Table of contents
+
+  * [Requirements](#requirements)
+  * [Installation](#installation)
+    + [Verify everything works](#verify-everything-works)
+  * [Usage](#usage)
+    + [Usage of available services](#usage-of-available-services)
+      - [Available services](#available-services)
+    + [Console Commands](#console-commands)
+      - [Available console commands](#available-console-commands)
+    + [Usage of SDK Models in templates](#usage-of-sdk-models-in-templates)
+  * [Quick-start](#quick-start)
+  * [Disclaimer for ExampleBundle (dev)](#disclaimer-for-examplebundle--dev-)
+  * [Testing](#testing)
+  * [Issues](#issues)
+  * [Contribute](#contribute)
+  * [License](#license)
 
+
+## Requirements
 Composer :
 To install the composer go to https://getcomposer.org/doc/00-intro.md
 
 ## Installation
-Create a new or open a symfony3 project and open it in any editor of choice
+Create a new Symfony project 
+```sh
+composer create-project symfony/skeleton <project-name>
+```
+or use an existing one (symfony >= 3.4)
 
-Open composer.json and add to the attribute require this line
+Navigate to the project's directory and run
+```sh
+composer config extra.symfony.allow-contrib true
+```
+This will allow symfony to use the recipes-contrib (via Symfony Flex) that will automate
+most of the configuration.
 
+To install the commercetools symfony-bundle open composer.json and add to the attribute 
+`require` the package `"commercetools/symfony-bundle"`
+and run `composer install` or directly run the following on the command line
 ```sh
 composer require commercetools/symfony-bundle
 ```
 
-Add the Commercetools Bundle to your application kernel
+Open `.env` or create a `.env.local` [1] file on root directory and edit the following 
+lines to add  your credentials. The credentials can be retrieved through 
+`Commercetools Merchant Center > Settings > Developer Settings`, while you create a new 
+API Client. Note that for security reasons you cannot retrieve the `CTP_CLIENT_SECRET` 
+for clients created in the past.
+
+```dotenv
+CTP_CLIENT_ID=<your client id>
+CTP_CLIENT_SECRET=<your client secret>
+CTP_PROJECT_KEY=<your project id>
+CTP_AUTH_URL=https://auth.commercetools.com or https://auth.commercetools.co
+CTP_API_URL=https://api.commercetools.com or https://api.commercetools.co
+CTP_SCOPES=<your desired scopes>
+```
+
+[1] To read more about the differences between the usage of .env and .env.local you can
+continue [here](https://symfony.com/doc/current/configuration/dot-env-changes.html) and/or
+[here](https://symfony.com/blog/new-in-symfony-4-2-define-env-vars-per-environment)
+
+### Verify everything works
+
+To verify that your configuration works, after adding your client credentials on the .env file
+as explained above, you may run on the command line:
```

Comment:
```suggestion
run the following on the command line:
```

```
bin/console commercetools:project-info
```

(sorry @nikossvnk – it doesn't play nicely with codeblocks, you'll need to do this manually too)